flake.nix

```diff
@@ -584,7 +590,8 @@
                   wasm-optimizer
                   xorriso
                   zlib.out
-                ];
+                  nix-tree
+                ] ++ docs-renders;
```

Comment:
`docs-renders` is a list of 4 elements.
Adding `docs-renders` to the head of the list will be faster.

Reply:
I think optimizes list with 32 length arrays so:) like LISPs do

Reply:
yes, you are right, there is the optimization 
```
$ NIX_SHOW_STATS=1 nix repl --expr '{a=[0] ++ [0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19];}' 
Welcome to Nix 2.10.3. Type :? for help.

Loading installable ''...
Added 1 variables.
nix-repl> a
[ 0 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 ]

nix-repl> {
  "cpuTime": 0.017752,
...

$ NIX_SHOW_STATS=1 nix repl --expr '{a=[0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19] ++ [0];}' 
Welcome to Nix 2.10.3. Type :? for help.

Loading installable ''...
Added 1 variables.
nix-repl> a
[ 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 0 ]

nix-repl> {
  "cpuTime": 0.013038,

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
$ NIX_SHOW_STATS=1 nix repl --expr '{a=[0] ++ [0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40];}' 
Welcome to Nix 2.10.3. Type :? for help.

Loading installable ''...
Added 1 variables.
nix-repl> a
[ 0 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 ]

nix-repl> {
  "cpuTime": 0.00693,
...

$ NIX_SHOW_STATS=1 nix repl --expr '{a=[0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40] ++ [0];}' 
Welcome to Nix 2.10.3. Type :? for help.

Loading installable ''...
Added 1 variables.
nix-repl> a
[ 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 0 ]

nix-repl> {
  "cpuTime": 0.008813,
 ...